Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.15.2-buster as builder
+WORKDIR /statsd/
+COPY . .
+RUN CGO_ENABLED=0 go build -ldflags '-extldflags "-static"' -o statsd
+
+FROM alpine:latest
+COPY --from=builder /statsd/statsd /statsd
```

Comment:
I suggest putting the binary in `/usr/local/bin/`, then it can be invoked without the leading `./`.

Also, I suggest naming the binary "statsdaemon", that's what it is named in the release tarballs (and by default when you just "go build"), and its command-line flags are not compatible with the classic nodejs statsd anyway.